SUPPORT.md

```diff
@@ -2,4 +2,8 @@
 
 Thanks for trying out Ark! We welcome all feedback, please consider joining our mailing list: 
 
-- [Mailing List](http://j.hept.io/ark-list)
\ No newline at end of file
+- [Mailing List](http://j.hept.io/ark-list)
+
+# Updates
+
+For the latest updates about us, follow [**@heptioark**](https://twitter.com/heptioark) on Twitter or visit the [**heptio blog!**](https://blog.heptio.com/)
```

Comment:
Please change `@heptioark` to `@HeptioArk` (capital H, capital A). Please also change `**heptio blog**` to `**Heptio blog**` (capital H in Heptio).

Once you've made these changes, please squash your commits down to a single commit and we should be able to merge. Thanks!

Reply:
@ncdc I have made the required changes to the code.

Reply:
@skriss @ncdc I made the commits correctly this time, but I am not able to resolve the sign-off!
Can you close this without that?

Reply:
@ssiddhantsharma we do need the signoff, and we'd like this to be squashed to a single commit. If you need assistance with either, please let us know.